Content.Server/Radio/Components/HeadsetComponent.cs

```diff
@@ -1,19 +1,44 @@
 using Content.Server.Radio.EntitySystems;
 using Content.Shared.Inventory;
-using Content.Shared.Radio;
-using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.Set;
+using Robust.Shared.Audio;
+using Robust.Shared.Containers;
 
 namespace Content.Server.Radio.Components;
-
 /// <summary>
 ///     This component relays radio messages to the parent entity's chat when equipped.
 /// </summary>
 [RegisterComponent]
 [Access(typeof(HeadsetSystem))]
 public sealed class HeadsetComponent : Component
 {
-    [DataField("channels", customTypeSerializer: typeof(PrototypeIdHashSetSerializer<RadioChannelPrototype>))]
-    public readonly HashSet<string> Channels = new() { "Common" };
+    [ViewVariables]
+    public HashSet<string> Channels = new()
+    {
+        "Common"
+    }; //Fills only by encryption chips in it
+
```

Comment:
This doesn't make sense? Because the common channel is still here and also now it's not being checked by the yaml linter and it's not being serialized.

Reply:
So should i just remove "Common" by default?
From headset and from keys?

Reply:
If you're making it use keys everything should be done by keys.

Reply:
> If you're making it use keys everything should be done by keys.

Already removed defaults of headset